src/components/results/Results.tsx

```diff
@@ -0,0 +1,35 @@
+import { ReactElement } from 'react';
+import Card from '../card/Card';
+import Pagination from '../pagination/Pagination';
+import styles from './Results.module.scss';
+import type { SearchResponseResults, Author } from '../../types/types';
+import type { DataResults } from './Results.types';
+
+function getArticlesCards(articles: SearchResponseResults[], authors: Author[]): ReactElement[] {
```

Comment:
Probably author should be part of the article already instead providing it separately. But that is not critical.